bosh/manifest.yml

```diff
@@ -310,7 +310,7 @@ instance_groups:
             auto_login: true
             client_id: ((oauth-proxy-client-id))
             client_secret: ((oauth-proxy-client-secret))
-            scopes: openid
+            scopes: openid,prometheus.support,concourse.pages,concourse.admin
```

Comment:
I'm curious about the addition of `concourse.pages` and `concourse.admin`. It makes sense to give the Pages team access, but didn't platform admins already have access. So why do we need to add `concourse.admin`?

Reply:
Yes, everyone had access actually. We're trying to break down the permissions to be more fine grain. Grafana to start, Prometheus next.

Reply:
Ah ok. And it looks like the `openid` scope is for UAA clients: https://docs.cloudfoundry.org/concepts/architecture/uaa.html#uaa-scopes

I'm just curious how everyone had access with the current configuration